src/EVM/UnitTest.hs

```diff
@@ -240,6 +244,7 @@ symRun opts@UnitTestOptions{..} vm (Sig testName types) = do
       let y = symFailure opts testName (fst cd) types x
       liftIO $ putStr $ "   \x1b[31m[FAIL]\x1b[0m " <> Text.unpack testName <> "\n" <> Text.unpack y
       pure False
+    where
```

Comment:
Looks like trailing `where`?
I am surprised this compiles :D

Reply:
Wow, yes, thanks, nice catch!

Reply:
Fixed.